src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.model;
+
+public class Fruit {
```

Comment:
Don't begin class or method implementation with empty line. Please remove the empty line at the beginning of the class definition.